plugins/com.google.cloud.tools.eclipse.preferences.test/src/com/google/cloud/tools/eclipse/preferences/PluginXmlTest.java

```diff
@@ -5,7 +5,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I'm fairly picky but a single indentation space in a copyright comment exceeds even my tolerance for adding junk to the commit history.